src/hooks/screen/useCreateChat.ts

```diff
@@ -71,6 +72,7 @@ const useCreateChat = () => {
       const initChannel = await SignedMessageRepo.createSignedChat(members);
       const chatData = createChannelJson(initChannel, selectedUser);
 
+      console.log('initChannel', JSON.stringify(initChannel));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive information like the entire `initChannel` object can pose a security risk if it contains any sensitive data. It's also not a good practice to leave console logs in production code as it can clutter the console output and make debugging harder. If this log is necessary for debugging, consider using a logging library that can be toggled for development/production environments.
```diff
-      console.log('initChannel', JSON.stringify(initChannel));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->